reporting/impl/src/main/java/org/jboss/windup/reporting/rules/generation/CreatePotentialIssuesReportRuleProvider.java

```diff
@@ -29,7 +28,7 @@
 /**
  * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
  */
-@RuleMetadata(phase = ReportGenerationPhase.class)
+@RuleMetadata(disabled = true, phase = ReportGenerationPhase.class)
```

Comment:
Yeah, I'm not seeing that error on master. I'm not sure what could be triggering it to break from this commit, though.